src/content/concepts/index.md

```diff
@@ -33,6 +34,8 @@ Every dependency is then processed and outputted into files called *bundles*, wh
 
 You can specify an entry point (or multiple entry points) by configuring the `entry` property in the [webpack configuration](/configuration).
 
+T> `entry` defaults to `./src`
```

Comment:
I don't think this needs to be a tip, you can just add it to the introductory paragraph and mention that `./src` is the default value for the `entry` property.